app/screens/settings-screen/settings-screen.tsx

```diff
@@ -139,11 +141,23 @@ export const SettingsScreen: React.FC = () => {
   }
 
   const [isContactModalVisible, setIsContactModalVisible] = React.useState(false)
-
   const toggleIsContactModalVisible = () => {
     setIsContactModalVisible(!isContactModalVisible)
   }
 
+  const rateUs = () => {
+    Rate.rate(ratingOptions, (success, errorMessage) => {
+      if (success) {
+        // this technically only tells us if the user successfully went to the Review Page.
+        // whether they actually did anything, we do not know.
+      }
+      if (errorMessage) {
+        // errorMessage comes from the native code. Useful for debugging, but probably not for users to view
+        console.error(`Rate.rate() error: ${errorMessage}`)
```

Comment:
we should probably also log the error in crashlytics so we have eyes there.
`crashlytics().recordError(err)`

Reply:
thanks, adding this.